.gitignore

```diff
@@ -127,6 +127,7 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+service_account/
```

Comment:
what is here?

Reply:
planning to store the service account json files here and then mount this directory to containers and then use the os.getenv to get service account details and pass to GCP services for auth

Reply:
just occurred to me, I might need to have a `service_account_template` directory similar to `.env` and `.env-template`

Reply:
good idea